src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -434,7 +434,7 @@ public async Task StopAsync(CancellationToken cancellationToken)
             var currentHost = ActiveHost;
             ActiveHost = null;
             Task stopTask = Orphan(currentHost, cancellationToken);
-            Task result = await Task.WhenAny(stopTask, Task.Delay(TimeSpan.FromSeconds(10), cancellationToken));
```

Comment:
This is removed in order to handle exception - "Host did not shutdown within allotted time". Because of Cancellation token signaled by ctrl+c in shutdown path, code is not waiting here for 10 seconds and throwing above warning.

Wait time increased from 10 sec to 15 sec to add grace period time (5 seconds).